### DISCUSSION TITLE
Exclude package from all rules

### DISCUSSION THREAD

**Discussion by inemtsev** [detekt/detekt#4625] [Q&A] [answered]
Hey guys,
I am trying to figure out how to exclude a package from testing. 
We have a module, which has a package **some-module/generated/many-things**
I have generated a detekt config file, but cannot find how to exclude an entire package (/generated/) from all rules. 
In docs I can only find how to exclude from specific rules.

Can you please recommend a solution?
Thanks

**Comment by cortinico** [comment, marked as answer]
You should be able to do so applying an `exclude` on the Gradle tasks for detekt. More on this here:
https://detekt.dev/gradle.html#leveraging-gradles-sourcetask---excluding-and-including-source-files

**Reply by inemtsev** [reply]
Thank you. This worked for me :)
